pkg/api/query/v1.go

```diff
@@ -142,6 +147,12 @@ func NewQueryAPI(
 		defaultInstantQueryMaxSourceResolution: defaultInstantQueryMaxSourceResolution,
 		defaultMetadataTimeRange:               defaultMetadataTimeRange,
 		disableCORS:                            disableCORS,
+
+		queryRangeHist: promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+			Name:    "query_range_duration_seconds",
```

Comment:
```suggestion
			Name:    "thanos_query_range_requested_timespan_duration_seconds",
```

To be consistent with the rest of the Thanos metrics this should have the `thanos_` prefix, I would also suggest we make this metric slightly more descriptive.

Reply:
Done.